_pages/install/integrations/opf-integration/opf-child-pages/1-open-payment-framework-in-spartacus.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Open Payment Framework in Spartacus
+---
+
+Open payment framework allows you to quickly integrate your preferred digital payment service providers in Spartacus, and removes the need to code, integrate, and deploy extensions to the SAP Commerce Cloud codebase.
```

Comment:
I do not think we can use the term Spartacus in the customer documentation. should we use the officially approved name?

Reply:
This term is converted to "composable storefront" when content is migrated to the help portal 🙂